Exports and imports all fields

Thenezzar opened the pull request:
Exports and imports all fields in the database for credentials except the relations

Thenezzar committed:
Export, Import all fields

Exports and imports all fields in for credentials except the relationships

wrongecho commented on agent/post/credential.php:
```diff
@@ -401,38 +402,41 @@
         $duplicate_count = 0;
         while(($column = fgetcsv($file, 1000, ",")) !== false){
             $duplicate_detect = 0;
-            // Name
```
Please don't remove these comments, they're useful!!

wrongecho commented:
Hello, thanks for the PR and sorry for the delay.

Could you update the downloadable import CSV template to match the new fields you've added please - `download_credentials_csv_template`?

Thenezzar committed:
Update credential_import.php

Removed these in the code as they point to specific Asset and Contact IDs but are probably only useful if you import into the same database where these already exist.
